data/templated/example.xml

```diff
@@ -61,4 +61,13 @@
          <value>11</value>
       </values>
    </template>
+    <template name="UpdateItemPrice">
+      <query><![CDATA[UPDATE item SET i_price = i_price + 1 WHERE i_price < ?]]></query>
+      <types>
+         <type>FLOAT</type>
```

Comment:
Might be good to have a few other test cases:
DELETE, INSERT, different data types, etc.